src/Compilers/CSharp/Portable/PublicAPI.Shipped.txt

```diff
@@ -3089,6 +3154,7 @@ Microsoft.CodeAnalysis.CSharp.SyntaxKind.UndefDirectiveTrivia = 8555 -> Microsof
 Microsoft.CodeAnalysis.CSharp.SyntaxKind.UndefKeyword = 8472 -> Microsoft.CodeAnalysis.CSharp.SyntaxKind
 Microsoft.CodeAnalysis.CSharp.SyntaxKind.UnderscoreToken = 8491 -> Microsoft.CodeAnalysis.CSharp.SyntaxKind
 Microsoft.CodeAnalysis.CSharp.SyntaxKind.UnknownAccessorDeclaration = 8900 -> Microsoft.CodeAnalysis.CSharp.SyntaxKind
+Microsoft.CodeAnalysis.CSharp.SyntaxKind.UnmanagedKeyword = 8446 -> Microsoft.CodeAnalysis.CSharp.SyntaxKind
```

Comment:
I'm surprised this keyword is new. I thought the unmanaged constraint introduced such a keyword. But it looks like 'where T : unmanaged' just parses as a special kind of type constraint? [sharplab](https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQxNGSAmIDUAfAAQCYBGAWACgCBmAAmNoGFaBvS2j+uggFloFkAPABUAfAAoAlLQDuAC0Rxaw2iFoBXAHaRNEAOZxMrdpwC+lU0A)

Reply:
Yep, it's just a special type constraint. But we needed a `managed` and `unmanaged` keyword for function pointers.

Reply:
In the case of the constraint, it could be that someone named their type `unmanaged` iirc.